index.js

```diff
@@ -1,5 +1,4 @@
 import Queue from 'yocto-queue';
-import {AsyncResource} from '#async_hooks';
```

Comment:
Since this import is being dropped, you'll probably want to remove the `#async_hooks` stub in `package.json` as well as `async-hooks-stub.js`

Reply:
Thanks!

 I removed the `#async_hooks` stub in `package.json`: https://github.com/sindresorhus/p-limit/pull/83/commits/b41d771c18c442457050c496b70541309103be3c